public/js/MainController.js

```diff
@@ -1,19 +1,19 @@
-/*global console, _*/
+    /* global _, app */
 
-(function () {
-    var app = angular.module("ChatApp", ["ngAnimate", "toastr"]);
-
-    app.controller("ChatController", function ($scope, $http, toastr) {
+    app.controller("ChatController", function ($scope, $http, $mdDialog, toastr) {
```

Comment:
Is this file only the ChatController? If it isn't the chat controller should be split out into a ChatController.js file. Or this needs to be renamed to ChatController.js